Drive/test/rover_drive_system_test.cpp

```diff
@@ -20,12 +20,7 @@ TEST_CASE("Drive system testing")
   StaticMemoryResource<1024> memory_resource;
   CanNetwork network(mock_can.get(), &memory_resource);
 
-  RmdX left_steer_motor(network, 0x141);
-  RmdX left_hub_motor(network, 0x142);
-  RmdX right_steer_motor(network, 0x143);
-  RmdX right_hub_motor(network, 0x144);
-  RmdX back_steer_motor(network, 0x145);
-  RmdX back_hub_motor(network, 0x146);
+  RmdX motor(network, 0x141);
```

Comment:
Did you verify this actually works as anticipated?

Reply:
Honestly no, I thought the fact that all the tests passed was good enough, how would I go about verifying this? I'm not too sure how I would do this.